src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -15,21 +15,21 @@ internal sealed partial class RuntimeModule : Module
         internal RuntimeModule() { throw new NotSupportedException(); }
 
         #region FCalls
```

Comment:
Update or remove?

Reply:
Moved FCalls next to their uses. Also, deleted some unused fields.